.github/workflows/test.yml

```diff
@@ -1,7 +1,7 @@
 name: test
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
Please use pull_request and not "pull_request_target".
This is a change we did during the immigration.